sql/telemetry_derived/deviations_v1/metadata.yaml

```diff
@@ -5,14 +5,7 @@ owners:
   - jmccrosky@mozilla.com
 labels:
   incremental: true
-  schedule: daily
   public_json: true
   public_bigquery: true
```

Comment:
We'll keep the public table and the public json data in GCS around for now since they are linked in the public blog post, but updates for the data will stop.